### PR TITLE
improve coverage of Instance Composer

### LINKED ISSUE #5339: Fill test cases wrote for Canvas
I'm aware that approach is not optimal, but jest introduced set of errors when working with jointjs library which cripples ability to write test there.

After most urgent improvements/fixes will be done I'll try to extract as much logic as possible to create coverage that won't include jointjs and even better react library as the UI interactions are one that causing troubles, and the rest will be done through cypress

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

I deleted test scenarios that was failing due to some issues with how responsive library is in the jest environment, But scenarios deleted are covered by the e2e scenarios

closes #5339 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

**Review by LukasStordeur — COMMENTED** [review]
I'm confused how removing the empty test cases that you prepared are improving the coverage.

**Comment by LukasStordeur** [comment]
> > I'm confused how removing the empty test cases that you prepared are improving the coverage.
> 
> Adding test for the standalone functions is improving coverage. I removed the test cases that were faulty, and the ones that are covered by the e2e, as it was discussed some time ago, due to the mentioned issues between jest and jointjs library.

My bad, looks like the diff was too big and I missed it entirely

**Comment by matborowczyk** [comment]
> My bad, looks like the diff was too big and I missed it entirely

No worries :D

File: src/UI/Components/Diagram/Canvas.test.tsx [hunk 1]
```diff
@@ -129,7 +128,11 @@ const deleteAndAssert = async (
   expect(updatedConnectors3).toHaveLength(assertionThree);
 };
 
-const createShapeWithNameAndId = async (shapeName: string, name, id) => {
+const createShapeWithNameAndId = async (
```
[the remark this answers is not in the document]
Comment (matborowczyk) [reply src/UI/Components/Diagram/Canvas.test.tsx:139]: Added

File: src/UI/Components/Diagram/helpers.test.ts
Comment (LukasStordeur) [inline src/UI/Components/Diagram/helpers.test.ts:1]: I also feel like a foreach loop for these tests would have been cleaner , like for example in the web-console\src\Slices\Agents\Data\getUrl.test.ts

File: src/UI/Components/Diagram/helpers.test.ts [hunk 2]
```diff
@@ -488,3 +525,819 @@ describe("shapesDataTransform", () => {
...
+      checkWhetherConnectionRulesAreExhausted(
+        [mockedMatchingInterService, mockedInterService],
+        RelationRuleNoLimit,
+        false,
+      ),
+    ).toBeFalsy();
+  });
+});
+
+Object.defineProperty(global.SVGElement.prototype, "getComputedTextLength", {
```
Comment (LukasStordeur) [inline src/UI/Components/Diagram/helpers.test.ts:905]: Can you add some comment why this is needed?

File: src/UI/Components/Diagram/helpers.test.ts
Comment (matborowczyk) [reply src/UI/Components/Diagram/helpers.test.ts:1]: I bundled the ones that had the identical flow

[the remark this answers is not in the document]
Comment (matborowczyk) [reply src/UI/Components/Diagram/helpers.test.ts:1]: done

**Review by LukasStordeur — APPROVED** [review]

**Pull request closed without merging**